go/cmd/dolt/commands/pull.go

```diff
@@ -92,6 +91,14 @@ func (cmd PullCmd) Exec(ctx context.Context, commandStr string, args []string, d
 		return HandleVErrAndExitCode(errhand.VerboseErrorFromError(err), usage)
 	}
 
+	// Call fetch; pass in remote name from pull spec
+	fetchExitCode := FetchCmd{}.Exec(ctx, "fetch", []string{pullSpec.Remote.Name}, dEnv)
```

Comment:
Don't do this, call the library method used by fetch

Might need to do a little cleanup there to make that possible